domain/lib/src/model/sharedspacedocument/work_group_folder.dart

```diff
@@ -33,6 +33,9 @@
 import 'package:domain/domain.dart';
 
 class WorkGroupFolder extends WorkGroupNode {
+
+  final List<TreeNode> listTreeNode;
```

Comment:
should be define in `WorkGroupNode`

Reply:
name is `treePath` to easier in parsing.

Reply:
ok